### ISSUE TITLE
UseUtilityClass false-positives

### ISSUE THREAD

**Issue by dgroup** [dgroup/arch4u-pmd#133]
### Summary
| Key            |  Value |
|:---------------|:------|
| Rule           | UseUtilityClass |
| arch4u version       | 0.1.1 |

### Your rule definition
```xml
default from pmd-java:6.44.0
```

### Describe the defect or false/positive rule violation

- [ ] Case 1: The typical spring boot main class

```java
@SpringBootApplication
public class Application {
    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }
}

```
